hepcrawl/spiders/magic_spider.py

```diff
@@ -24,22 +24,25 @@
 class MagicSpider(XMLFeedSpider):
 
     """MAGIC crawler
-    Scrapes theses metadata from MAGIC telescope web page.
-    https://magic.mpp.mpg.de/backend/publications/thesis
 
-    1. `parse_node` will get thesis title, author and date from the listing.
-    2. If link to the splash page exists, `scrape_for_pdf` will try to fetch
+    Scrapes theses metadata from `MAGIC telescope web page`_.
+
+    1. ``MagicSpider.parse_node`` will get thesis title, author and date from the listing.
+    2. If link to the splash page exists, ``MagicSpider.scrape_for_pdf`` will try to fetch
        the pdf link, abstract, and authors.
-    3. `build_item` will build the HEPRecord.
+    3. ``MagicSpider.build_item`` will build the ``HEPRecord``.
+
+    Examples:
+        Default example::
```

Comment:
and here (instead of default, say basic, default means the one that is used if none passed, makes 'default example' makes no sense to me, unless you are reusing the same example for any project that does not have one :/ )